.github/workflows/on_push.yml

```diff
@@ -50,7 +50,8 @@ jobs:
         clickhouse-version:
           - '22.3'
           - '22.8'
-          - '22.9'
+          - '22.10'
+          - '22.11'
```

Comment:
+ 22.12?

Reply:
well that was `latest` when I started this PR :)